### DISCUSSION TITLE
switch wallet sync

### DISCUSSION THREAD

**Discussion by lpf763827726** [Chia-Network/chia-blockchain#7011] [Ideas]
if i open with gui, i have two wallet, when i open the first wallet and complete synchronization, but if i switch second wallet, i have to sync again, they are in the same net, why can't i sync together? always have to sync again, it have a little waste time, how about try to sync all wallet in `chia start wallet` same time in gui?
because i want to test the pools, i want to check the TXCH and test receive payouts, i need to switch the wallet, and i need to wait a little than i can check wallet and switch the wallet

**Comment by loppefaaret** [comment]
currently, the wallet is only able to handle a single key at a time - we do have plans to at some point expanding this function to be able to sync mutliple wallets at the same time

**Reply by lpf763827726** [reply]
ok, I'm looking forward to this feature :)
